inference-engine/samples/benchmark_app/benchmark_app.hpp

```diff
@@ -145,8 +145,14 @@ static const char dump_config_message[] =
 #endif
 
 static const char shape_message[] =
-    "Optional. Set shape for input. For example, \"input1[1,3,224,224],input2[1,4]\" or "
-    "\"[1,3,224,224]\""
+    "Optional. Set shape for network input. For example, \"input1[1,3,224,224],input2[1,4]\" or \"[1,3,224,224]\""
+    " in case of one input size. This parameter affect model Parameter shape, can be dynamic. For dynamic dimesions"
+    " use symbol `?`.";
```

Comment:
usually `-1` is used

Reply:
It already supports both  ? and -1. Messgae text is fixed.